pdfs/src/Pdf/FromLatex.hs

```diff
@@ -0,0 +1,206 @@
+
+module Pdf.FromLatex where
+
+import qualified Data.ByteString as BS
+import Path.IO
+import Path
+import System.Directory
+import System.IO.Temp
+import System.Process.Typed
+
+escapeInLatex :: Text -> Text
+escapeInLatex text = text
+
+type Locale = Text
+
+data Date = Date
+    { year  :: Int
+    , month :: Int
+    , day   :: Int
+    }
+
+dateToString :: Locale -> Date -> Text
+dateToString _ (Date y m d) = show d <> "." <> show m <> "." <> show y
+
+data StudentInfo = StudentInfo
+    { siName        :: Text
+    , siDateOfBirth :: Date
+    , siSections    :: [Section]
+    }
```

Comment:
Hm, it is called `StudentInfo`, but contains the whole certificate, is it intended? :thinking:

Reply:
Nope, I'll use @flyingleafe 's type anyway